packages/react/src/domain/services/JWTService.ts

```diff
@@ -0,0 +1,9 @@
+export class JWTService {
+  static decodeToken(token: string): {
+    rate: number;
+    exp: number;
+    iat: number;
+  } {
+    return JSON.parse(Buffer.from(token?.split('.')[1], 'base64').toString());
+  }
+}
```

Comment:
I think we removed this and used a jwt decode library with mapping to types. Might be best to use that instead as not all decode tokens are going to have this type. For example swap token will not have a rate

Reply:
agree we should just use the same lib we use in dash, and BE